Optimization Online.js

```diff
@@ -0,0 +1,418 @@
+{
+	"translatorID": "0ee623f9-0a5c-4f6a-a5e1-10e2feaa16a7",
+	"label": "Optimization Online",
+	"creator": "Pascal Quach",
+	"target": "^https?://optimization-online\\.org/",
+	"minVersion": "6.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2023-07-15 13:50:06"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2023 Pascal Quach <pascal.quach@centralesupelec.fr>
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) { // eslint-disable-line no-unused-vars
+	if (doc.body.classList.contains('single-post')) {
+		return 'preprint';
```

Comment:
@AbeJellinek @dstillman  -- at one point do we decide that it's OK if translators don't work for versions anymore? Preprint was introduced in 6.0, almost 18 month ago, I feel like it's fine to just return this as a type now (as is done here) rather than working with the alternates for pre 6.0 versions, thoughts? 
Since this will come up repeatedly with new item types, it'd be good to have an approximate policy/guideline for this.

Reply:
Yes, totally fine to break 5.0 compatibility at this point.